packages/replayio/src/utils/recordings/getRecordingUnusableReason.ts

```diff
@@ -0,0 +1,20 @@
+import { getRecordings } from "./getRecordings";
+import { LocalRecording } from "./types";
+
+export function getRecordingUnusableReason(prevRecordings: LocalRecording[]) {
+  const newCrashedRecordings = getRecordings().filter(recording => {
+    if (!prevRecordings.some(({ id }) => id === recording.id)) {
+      return recording.recordingStatus === "crashed";
+    }
+  });
+
+  if (newCrashedRecordings) {
+    // Look for the most recent crashed recording; that is most likely to be related
+    for (let index = newCrashedRecordings.length - 1; index >= 0; index--) {
+      const recording = newCrashedRecordings[index];
+      if (recording.unusableReason) {
+        return recording.unusableReason;
+      }
+    }
```

Comment:
nit:
```suggestion
    return newCrashedRecordings.findLast(recording => recording.unusableReason)?.unusableReason;
```

Reply:
In a sense, we don't target ES2021 here but node@18 and `findLast` is there. According to https://node.green/ there is nothing substantial in node@18 that would be missing from ES2022 and ES2023 - so we could bump the compilation target to ES2023.

I was changing that compilation target already in https://github.com/replayio/replay-cli/pull/369 and I think I just picked the tsconfig values by dates and I didn't consult the compatibility table to see if I could bump it any higher.

I'm fine either way though

Reply:
https://github.com/replayio/replay-cli/pull/441

Reply:
Can't get to this with the ES2022 bump but oh well :)

Reply:
It works on my machine now 
<img width="1173" alt="findLast being successfully resolved by VS Code" src="https://github.com/replayio/replay-cli/assets/9800850/3fcce268-a18d-41f1-9b5d-e131f901e06e">

And it only works in `packages/replayio` (which is what we want). Other packages don't see this.

1. Could you recheck what TS version you use in your IDE? I always open command palette in projects and "Select TypeScript version..." to use the one provided by the workspace (otherwise VS Code sticks to its builtin version)
2. Does `yarn boostrap` in the root work OK with this change?

Reply:
My editor wasn't the one complaining, it was `yarn build` – but now it seems to be working 🤷🏼  Maybe I forgot to `yarn install` again after switching branches.